recipe/meta.yaml

```
@@ -84,20 +69,19 @@ requirements:
     - libtasn1  # [osx]
     - exiv2
     # Qt stack.
-    - qt
+    - qt-main
     - pyqt
+    - pyqt5-sip
+    - pyqt-builder =1.13
     - qtwebkit
     - pyqtwebkit
     - qscintilla2
     - qca
     - qjson
     - qwt
-    - qwtpolar
     - qtkeychain
-    - qtlocation
-    # included in win-64 Qt already but needed for linux/osx
-    - qtserialport  # [unix]
     # GDAL stack.
+    - libgdal
     - gdal
```

Best to pin GDAL here so that `gdal >=3.5,<3.6.0a0` is specified.
```suggestion
    - {{ pin_compatible('gdal', max_pin='x.x') }}
```